src/features/comments/commentsSlice.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-cycle
+import {
+  createComment,
+  deleteComment,
+  getPostComments,
+} from '../../api/comments';
+import { Comment } from '../../types/Comment';
+
+const initialState = {
+  comments: [] as Comment[],
+  hasError: false,
+  loaded: true,
+};
+
+export const setComments = createAsyncThunk(
+  'comments/set',
+  (postId: number) => {
+    return getPostComments(postId);
+  },
+);
+
+export const addComment = createAsyncThunk(
+  'comments/add',
+  async (comment: Omit<Comment, 'id'>) => {
+    const value = await createComment(comment);
+
+    return value;
+  },
+);
+
+export const removeComment = createAsyncThunk(
+  'comments/delete',
+  (commentId: number) => {
+    return deleteComment(commentId);
+  },
+);
+
+export const commentsSLice = createSlice({
```

Comment:
The slice name `commentsSLice` should be renamed to `commentsSlice` to follow JavaScript naming conventions.